CANGAMetricsDriver.py

```diff
@@ -172,6 +172,26 @@ def loadMeshData(mesh_file, mesh_config, SpectralElement):
               endt = time.time()
               print('Time to read SCRIP mesh info (sec): ', endt - start)
               
+       elif mesh_config == 4:
+              numEdges = 'num_nod_per_el1'
```

Comment:
Is this correct to use the num_nod_per_el from level 1 and the num_el_in_blk from level 0 together?

Reply:
numEdges will be removed from the metrics driver.